### PR TITLE
[framework-release] update yaml

### PR DESCRIPTION
**Pull request by rahxephon89**

### CODE REVIEW

**Review by msmouse — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@ proposals:
       - FeatureFlag:
           enabled:
             - bulletproofs_natives
-  - name: enable_storage_deletion_refund
+  - name: step_6_enable_storage_deletion_refund
     metadata:
       title: "Enable Storage Deletion Refund"
       description: "AIP-32: Storage Deletion Refund"
```
Comment (thepomeranian) [inline aptos-move/aptos-release-builder/data/release.yaml:66]: @msmouse can you write... a better description for storage deletion refund?

Comment (msmouse) [inline aptos-move/aptos-release-builder/data/release.yaml:66]:
```suggestion
      description: "AIP-32: Storage Deletion Refund. A refund for the slot allocation fee is issued on the deletion of a global state item."
```

Comment (rahxephon89) [reply aptos-move/aptos-release-builder/data/release.yaml]: done

**Review by perryjrandall — APPROVED** [review]
Lgtm please write a description or test plan

**Pull request merged**